geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/ExportCommand.scala

```diff
@@ -27,25 +28,27 @@ import org.opengis.filter.Filter
 import scala.util.{Failure, Success, Try}
 
 class ExportCommand(parent: JCommander) extends CommandWithCatalog(parent) with LazyLogging {
+
   override val command = "export"
   override val params = new ExportParameters
 
   override def execute() = {
     val fmt = Formats.fromString(params.format)
     val features = getFeatureCollection(fmt)
+    lazy val avroCompression = Option(params.gzip).map(_.toInt).getOrElse(Deflater.DEFAULT_COMPRESSION)
```

Comment:
yeah, i checked - default == -1, which currently corresponds to level 6, but may change (although seems unlikely)